### ISSUE TITLE
error 256 no such file or directory

### ISSUE THREAD

**Issue by jxchong** [fengcong3/HAPPE#6]
HAPPE keeps complaining "no such file or directory" but weirdly it is complaining about how it cannot find the OUTPUT file that would be created when I provide the --output argument

Here's the error using your example files. I'm on a Mac running 13.5.2 and using zsh as my shell (although I tried in bash too).
```
% HAPPE \
-g happe.config \
-v SEVIR.592.SNP.ann.allele2.part.vcf.gz \
-i sample_inf.tsv \
-c sample.color.txt \
-k small_example.list \
-o SEVIR_5G085400v2

[2023-10-02 16:31:25,816] - HAPPE - INFO - deal args.
[2023-10-02 16:31:25,818] - HAPPE - INFO - Filter samples and variants.
ln: SEVIR_5G085400v2.vcf.gz: No such file or directory
[2023-10-02 16:31:25,842] - HAPPE - ERROR - Filter samples and variants. -- retrun code:256
```

**Comment by jxchong** [comment]
FYI, I installed this on my cluster just to try a different system but receive a the same because that file does not exist.

```
$ HAPPE \
> -g happe.config \
> --gzvcf SEVIR.592.SNP.ann.allele2.part.vcf.gz \
> -i sample_inf.tsv \
> -c sample.color.txt \
> -k small_example.list \
> -o SEVIR_5G085400v2
[2023-10-03 15:46:00,257] - HAPPE - INFO - deal args.
[2023-10-03 15:46:00,257] - HAPPE - INFO - Filter samples and variants.
ln: failed to create symbolic link ‘SEVIR_5G085400v2.vcf.gz’: File exists
[2023-10-03 15:46:00,261] - HAPPE - ERROR - Filter samples and variants. -- retrun code:256
```

**Comment by fengcong3** [comment]
Thanks for give HAPPE a try!

1. I'm sorry I don't have a mac environment and can't test the first error.
2. For the second error, the log you provided indicates that the file already exists. You can either delete the `SEVIR_5G085400v2.vcf.gz` file or change the execution directory. 

I would also recommend that you provide the -r option, as I wrote in the example.

**Comment by jxchong** [comment]
In both cases the file does NOT exist, that's the weird thing. There is no such file by that name in the folder, so I don't know why it's complaining about it. Similarly in the first scenario, it's complaining about the output file not existing, when of course it doesn't exist because the program hasn't been run yet. (based on the error message, there might be a stage at which you attempt to create a link to the file with `ln` ?)

**Comment by fengcong3** [comment]
Yes, if you don't provide the `-r` option or the `-s` option, it will just create a softlink to the input file. I suggest you try specifying a specific chromosome region such as `-r 5:6847970-6850236` in my example, and then see if you get any errors.
